modules/core/src/core/animation-loop.js

```diff
@@ -66,6 +66,8 @@ export default class AnimationLoop {
     this.cpuTime = 0;
     this.gpuTime = 0;
 
+    this.canvasDataURLResolver = null;
```

Comment:
Maybe this should be a private/underscore method? 

Naming of function is usually verbNoun, not noun: `_resolveCanvasDataUrl`. 

A name that was a noun would typically represent an object instance in my mind...